javascript/step_3.html

```diff
@@ -1,13 +1,13 @@
 <div class="instructions">
     <h2>Routes and functions</h2>
-    <p>Next we need to declare some routes. These are endpoints that our application will be exposing when it's up and
-        running, and that will be called by the Pokerwars Engine! This Engine is where all the magic happens. It keeps
-        track of all the tournaments and calls each bot for the next action. For now, let's not worry about that and
+    <p>Next we need to declare some routes. These are endpoints that our bot will be exposing when it's up and
+        running, and that will be called by the pokerwars Engine! This Engine is where all the magic happens. It keeps
+        track of all the poker tournaments and calls each bot for the next action. For now, let's not worry about that and
         let's focus on how to create routes.
     </p>
     <p>In Express, we can create a new route using
         <i>app.route()</i>. In the code, we went ahead and created a route for the <i>play</i> action for you. This
-        is the endpoint the Engine will call (with a POST request) during the tournament to ask you for your next action
+        is the endpoint the Engine will call (with a POST request) during the tournament to ask you for your next poker action (fold, call, bet or raise)
```

Comment:
this line is quite long, you can break it (sorry just cosmetics)

Reply:
reason I left it long is that intellij fucks everything up if you try to break it down, I would need to reformat the whole file unfortunately